src/js/ScrollArea.jsx

```diff
@@ -89,6 +89,14 @@ export default class ScrollArea extends React.Component {
         this.setSizesToState();
     }
 
+    get touchEventProps() {
```

Comment:
TO be more consistent please not use getters. The best solution will be to just do this in props definitions:
```
{...(this.props.preventScrollByTouch ? {} : {
        onTouchStart: this.handleTouchStart.bind(this),
        onTouchMove: this.handleTouchMove.bind(this),
        onTouchEnd: this.handleTouchEnd.bind(this)
      })
}
```